lib/table.py

```diff
@@ -110,7 +112,11 @@ def combine_on_delimiter(self, line_as_list):
         """Combine a list of values into a line of csv data"""
         dialect = csv.excel
         dialect.escapechar = "\\"
-        writer_file =  io.BytesIO()
+        if sys.version_info >= (3, 0):
+            writer_file =  io.StringIO()
+        else:
+            writer_file =  io.BytesIO()
```

Comment:
@goelakash - can you walk me through the difference in `io.StringIO()` and `io.BytesIO()` between Python 2 & 3?